Demo/Demo Base/DemoAppDelegate.swift

```diff
@@ -105,13 +106,7 @@ class DemoAppDelegate: UIResponder, UIApplicationDelegate {
                 UserDefaults.standard.set(testIntegration, forKey: "BraintreeDemoSettingsAuthorizationOverride")
             }
         }
-        
-        if testArguments.contains("-BadUrlScheme") {
```

Comment:
This was used to test an incorrect URL scheme being set for the PayPal flow. Since `BTPayPalDriver` no longer requires a custom URL scheme, we removed the test and this flag.